ci-operator/step-registry/ipi/conf/aws/route53/private-hosted-zone/ipi-conf-aws-route53-private-hosted-zone-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: ipi-conf-aws-route53-private-hosted-zone
```

Comment:
per our existing workflow, we do not need to create a chain for updating configuration, like [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-provision-chain.yaml#L10-L14): 

```
    - ref: ipi-conf
    - ref: ipi-conf-mirror
    - ref: ipi-conf-aws-custom-vpc
    - ref: ipi-conf-manual-creds
    - ref: ipi-conf-aws
    - ref: ipi-install-monitoringpvc
    - ref: proxy-config-generate
    - ref: ipi-install-install-aws
```
we flatten all steps in `*-provision` chain, this will be more clear what's updated to the configuration.

Reply:
@jianlinliu suggested [here](https://github.com/openshift/release/pull/27656#discussion_r845768816) to move  `ipi-conf-aws` behind the `ipi-conf-aws-route53-private-hosted-zone`.  @yunjiang29 @jianlinliu please let me know where I should keep these configurations.

Reply:
I mean we should delete chain `ipi-conf-aws-route53-private-hosted-zone`, and keep its step (`ref`) only, and also per Jianlin's comment, the config should be like:
```
chain:
  as: cucushift-installer-rehearse-...-provision
  steps:
...
    - ref: ipi-conf
    - ref: ipi-conf-aws-custom-vpc
    - ref: ipi-conf-aws-route53-private-hosted-zone
    - ref: ipi-conf-aws
...
```

Reply:
@yunjiang29 Suggestions have been implemented.